src/Administration/Resources/app/administration/src/module/sw-plugin/component/sw-plugin-table-entry/sw-plugin-table-entry.html.twig

```diff
@@ -2,7 +2,8 @@
     <div class="sw-plugin-table-entry">
         {% block sw_plugin_table_entry_icon %}
             <img v-if="icon" :src="'data:image/png;base64, ' + icon"/>
```

Comment:
would it makes sense to put the whole img thing into a computed property?

Reply:
Like that idea. This way: https://github.com/shopware/platform/pull/690/commits/1d58a6487ab720c24138f7fcd8c79cebd0ff7923
?

Reply:
looks good to me, but I'm not a JS pro :smile: